.circleci/config.yml

```diff
@@ -43,9 +43,10 @@ jobs:
   release:
     executor: node
     steps:
+      - setup
       - run:
           name: Create a new release with proper changelog
-          command: npx install semantic-release
+          command: npx semantic-release
```

Comment:
Can you explain this command a little more? I'm not sure what it does

Reply:
This uses the ENV variables from CircleCI to perform a package release on Github/NPM. We currently have the proper authentication env vars setup on CircleCI and that's how it has the permissions to work